modules/andes-core/broker/src/main/java/org/wso2/andes/server/information/management/QueueManagementInformationMBean.java

```diff
@@ -930,14 +929,72 @@ public int getSubscriptionCount( String queueName){
     }
 
     /**
+     * Method to construct message body of a single message.
+     *
+     * @param bodySize   Original content size of the message
+     * @param amqMessage AMQMessage
+     * @return Message content and last position of written data as an AndesMessagePart
+     * @throws MBeanException
+     */
+    private AndesMessagePart constructContent(int bodySize, AMQMessage amqMessage) throws MBeanException {
+
+        List<Byte> messageContent = new ArrayList<Byte>();
+        java.nio.ByteBuffer buffer = java.nio.ByteBuffer.allocate(bodySize);
```

Comment:
Used this to import two classes with the same name. java.nio.ByteBuffer is only using at this point.